test/batch_node_online.jl

```diff
@@ -384,7 +384,7 @@ end
         @test status(worker_job) == AWSBatch.SUCCEEDED
 
         manager_log = log_messages(manager_job)
-        worker_log = log_messages(worker_job)
+        worker_log = log_messages(worker_job; retries=0)
```

Comment:
Any reason to set `retries=0` here?

Reply:
The worker job isn't supposed to have any log events so not retrying just speeds things up a little.

Reply:
Ah gotcha!